Dockerfile

```diff
@@ -145,6 +145,7 @@ LABEL maintainer="Phylum, Inc. <engineering@phylum.io>"
 LABEL org.opencontainers.image.source="https://github.com/phylum-dev/phylum-ci"
 
 ENV PHYLUM_VENV="/opt/venv"
+ENV GLOBAL_HOME="/usr/local"
```

Comment:
This is extremely misleading, it has nothing to do with a home directory anymore.

I'd recommend using `TARGET_DIR`/`INSTALL_DIR` or anything better you can come up with that doesn't misappropriate the term "home".